simple_costmap:
  start_time: 5
  padding: 2 # number of grid cells to pad the c_scpace with